### PR TITLE
[Platform] Collocation pages

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Refactor of sidebar
- Refactor of layout
- Collocation landing pages

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?

- View pages in deploy preview

#### What are the relevant tickets?

- [AN-269]

#### Screenshots
image

image

[AN-269]: https://airqoteam.atlassian.net/browse/AN-269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: platform/src/common/components/SideBar/SideBarItem.jsx [hunk 1]
```diff
@@ -5,49 +5,48 @@ import ArrowDropDownIcon from '@/icons/arrow_drop_down.svg';
...
 
   return (
     <div
-      className={`${
-        toggleDropdown && 'bg-[#D9D9D9] rounded-lg'
-      } cursor-pointer`}
+      className={`cursor-pointer ${
+        toggleDropdown && 'bg-sidebar-blue rounded'
+      }`}
     >
```
Comment (Codebmk) [inline platform/src/common/components/SideBar/SideBarItem.jsx:29]: NIce refactoring @FaithDaka

File: platform/.vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "dbaeumer.vscode-eslint",
+    "bradlc.vscode-tailwindcss",
+    "github.vscode-pull-request-github",
+    "firsttris.vscode-jest-runner",
+    "christian-kohler.npm-intellisense",
+    "wakatime.vscode-wakatime",
+    "willstakayama.vscode-nextjs-snippets",
+    "dsznajder.es7-react-js-snippets"
+  ]
+}
```
Comment (Codebmk) [inline platform/.vscode/extensions.json:13]: Curious about these configurations. Do we need them or they sneaked into the PR by mistake? @FaithDaka

**Review by Codebmk — CHANGES_REQUESTED** [review]
Thanks for the work @FaithDaka. The collocate page overflows. Please look into that.
Screenshot from 2023-01-19 15-04-38

File: platform/.vscode/extensions.json (hunk 2 above)
Comment (FaithDaka) [reply platform/.vscode/extensions.json:13]: They are required VS Code extensions that maximise efficiency of a=the dev team. m=Makes it easy for new collaborators to plug in. Especially the snippets and intellisense

**Comment by Codebmk** [comment]
> Thanks for the work @FaithDaka. The collocate page overflows. Please look into that. Screenshot from 2023-01-19 15-04-38


Overflow solved. 
Screenshot from 2023-02-01 21-35-58

**Review by Codebmk — APPROVED** [review]
LGTM

File: platform/.vscode/extensions.json (hunk 2 above)
Comment (Codebmk) [reply platform/.vscode/extensions.json:13]: Alright thanks

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @FaithDaka 
LGTM

**Review by Baalmart — APPROVED** [review]
Thanks @FaithDaka

**Pull request merged**